ts/components/Pinpad/index.tsx

```diff
@@ -206,7 +207,9 @@ class Pinpad extends React.PureComponent<Props, State> {
     );
   };
 
-  public clear = () => this.setState({ value: "" });
```

Comment:
**_PinScreen_** uses the **_clear_** method (this is because it is public)
You should rename these references too and check if everything is working on

Reply:
reviewed within https://github.com/teamdigitale/io-app/pull/1301/commits/1d7c67f9539fffe6819568aa79d9cad70781935c